pkg/internal/checkup/trex-client/trex-client_test.go

```diff
@@ -0,0 +1,133 @@
+			"                total_tx_L2  : 0 bps                          \nversion      : STL @ v2.87                                total" +
+			"_tx_L1  : 0 bps                          \ncpu_util.    : 0.0% @ 6 cores (6 per dual port)           total_rx     : 9.42 Kbps  " +
+			"                    \nrx_cpu_util. : 0.0% / 0 pps                               total_pps    : 0 pps                          \n" +
+			"async_util.  : 0% / 0 bps                                 drop_rate    : 0 bps                          \ntotal_cps.   : 0 cps " +
+			"                                     queue_full   : 0 pkts                         \n\ntrex>Shutting down RPC client\n"
+		portStatsOutput = "Using 'python3' as Python interpeter\n\n\n-=TRex Console v3.0=-\n\nType 'help' or '?' for supported actions\n\ntr" +
+			"ex>\nverbose set to on\n\n\n\n[verbose] Sending Request To Server:\n\n[\n    {\n        \"id\": \"n5thi6a2\",\n        \"jsonrp" +
+			"c\": \"2.0\",\n        \"method\": \"get_port_stats\",\n        \"params\": {\n            \"api_h\": \"kGQyN8we\",\n          " +
+			"  \"port_id\": 0\n        }\n    }\n]\n\n\n\n[verbose] Server Response:\n\n{\n    \"id\": \"n5thi6a2\",\n    \"jsonrpc\": \"2.0" +
+			"\",\n    \"result\": {\n        \"ibytes\": 68625,\n        \"ierrors\": 10,\n        \"ipackets\": 893,\n        \"m_cpu_util\"" +
+			": 11.0,\n        \"m_total_rx_bps\": 6630.8935546875,\n        \"m_total_rx_pps\": 9.337542533874512,\n        \"m_total_tx_bps" +
+			"\": 12.0,\n        \"m_total_tx_pps\": 13.0,\n        \"obytes\": 14,\n        \"oerrors\": 15,\n        \"opackets\": 16\n    " +
+			"}\n}\n\nPort Statistics\n\n   port    |         0         \n-----------+------------------\nowner      |              root \nli" +
+			"nk       |                UP \nstate      |              IDLE \nspeed      |           10 Gb/s \nCPU util.  |              0.0%" +
+			" \n--         |                   \nTx bps L2  |             0 bps \nTx bps L1  |             0 bps \nTx pps     |             " +
+			"0 pps \nLine Util. |               0 % \n---        |                   \nRx bps     |         6.63 Kbps \nRx pps     |        " +
+			"  9.34 pps \n----       |                   \nopackets   |                 0 \nipackets   |                 0 \nobytes     |   " +
+			"              0 \nibytes     |                 0 \ntx-pkts    |            0 pkts \nrx-pkts    |            0 pkts \ntx-bytes  " +
+			" |               0 B \nrx-bytes   |               0 B \n-----      |                   \noerrors    |                 0 \nierro" +
+			"rs    |                 0 \n\ntrex>Shutting down RPC client"
+	)
+
+	switch strings.Join(command, " ") {
+	case composeTrexConsoleRequest("verbose on;stats -g"):
+		return globalStatsOutput, "", nil
+	case composeTrexConsoleRequest("verbose on;stats --port 0 -p"):
+		return portStatsOutput, "", nil
+	default:
+		return "", "", fmt.Errorf("unknown command: %v", command)
+	}
```

Comment:
Is it possible to please move the call to `composeTrexConsoleRequest` to outside of the `switch` statement?
I think it could improve readability.

Reply:
DONE